include/mata/nfa.hh

```diff
@@ -740,10 +632,10 @@ public:
      * @param state_from[in] Source state for transitions to get.
      * @return List of transitions leading from @p state_from.
      */
-    const TransitionList& get_transitions_from(const State state_from) const
+    const Moves& get_moves_from(const State state_from) const
```

Comment:
I wonder, why we don't use `pre`/`post` notation, that is clear for automata-theory guys?

Reply:
We could and probably should. Furthermore, have we not already agreed on using `Post` for the set of `Move` for a single source state?

Reply:
this will be part of another PR refactoring data structure in the way we discussed on the last meeting.